### PR TITLE
Code enhanced - Standardize field name #14697

### PR DESCRIPTION
**Pull request by aspangaro**

### CODE REVIEW

File: htdocs/comm/propal/class/propal.class.php [hunk 1]
```diff
@@ -185,6 +185,12 @@ class Propal extends CommonObject
 	 */
 	public $total;
 
+	public $total_ht = 0;
```
Comment (frederic34) [inline htdocs/comm/propal/class/propal.class.php:188]: these def are already in commonobject class

**Comment by eldy** [comment]
travis ci return some errors. It seems some code is still using the field with name "tva" somewhere.
May be in code to create an order from a proposal ?
Or may be inthe method update_price() of CommonObject.

**Comment by frederic34** [comment]
@aspangaro look in commonobject.class.php https://github.com/Dolibarr/dolibarr/blob/eeedbccc711a57fb1d09ac6619aeacb7fdbefe6c/htdocs/core/class/commonobject.class.php#L3233

```
			if ($this->element == 'propal') {
				$fieldtva = 'total_tva';
				$fieldttc = 'total';
			}
```
perhaps you can also migrate total to total_ttc

**Comment by aspangaro** [comment]
Thanks @eldy & @frederic34 

You're right, I forgot a field vat in commonobject

_perhaps you can also migrate total to total_ttc_
😃 A PR one by one

**Comment by eldy** [comment]
Yes please a PR one by one, above all on such reegeniering PR that can create important regression.

**Pull request merged**